SR2023/2022-11-23-virtual-december-tech-day.md

```diff
@@ -0,0 +1,30 @@
+---
+to: SR2023 teams who were not coming to the London Tech Day
+subject: Virtual Tech Day
+---
+
+Hello,
+
+We will be hosting a [virtual Tech Day on the 3rd December][virtual-tech-day] in
+our [Discord](https://studentrobotics.org/docs/team_admin/discord) server.
+
+Tech Days are opportunities for teams to spend a whole day working on their
+robot with lots of help available. They’re also an opportunity to see how other
+teams are doing or get more direct help with your robots.
```

Comment:
[nit] we're mixing "their" and "your" here

```suggestion
teams are doing or get more direct help with their robots.
```

Reply:
Think this is actually fine, as discussed.